src/java/htsjdk/variant/variantcontext/Allele.java

```diff
@@ -189,6 +190,7 @@ protected Allele(final Allele allele, final boolean ignoreRefState) {
     private final static Allele ALT_T = new Allele("T", false);
     private final static Allele REF_N = new Allele("N", true);
     private final static Allele ALT_N = new Allele("N", false);
+    private final static Allele SPAN_DEL = new Allele(VCFConstants.SPANNING_DELETION_ALLELE, false);
     public final static Allele NO_CALL = new Allele(NO_CALL_STRING, false);
```

Comment:
I think this should be public, not private.  It's okay for others to use.  And you will want to do so in CombineGVCFs and GenotypeGVCFs.